engine/src/main/resources/assets/i18n/menu_fa.lang

```diff
@@ -0,0 +1,244 @@
+    "gameplay": "محتویات نمایش بازی",
+    "gameplay-info": "can click 'modules' to customize further",
+    "host-multiplayer-game": "بازی میزبان",
+    "ingame-menu-title": "منوی در حین بازی",
+    "input-settings": "ورودی",
+    "input-settings-title": "تنظیمات ورودی",
+    "invert-mouse": "موشواره وارونه",
+    "invert-x": "معکوس ایکس اکسیس",
+    "invert-y": "معکوس وای اکسیس",
+    "invert-z": "معکوس زد اکسیس",
+    "join-game-online": "پیوستن به بازی",
+    "join-server-refresh": "تازه کردن",
+    "join-server-action": "پیوستن",
+    "join-server-requested": "تقاضا شده",
+    "join-server-title": "اتصال به سرور",
+    "light-shafts": "پرتوهای نور",
+    "listed-servers": "لیست شده",
+    "load-game": "ذخیره گذاری",
+    "menu-animations": "فهرست فیلمهای نقاشی شده",
+    "missing-name-message": "شما باید اسم را وارد کنید",
+    "module-dependencies-empty": "This module has no dependencies",
+    "module-dependencies-exist": "توابع طرحها",
+    "module-filter": "فیلتر",
+    "module-list": "طرح ها",
+    "module-name": "اسم",
+    "module-status": "وضعیت",
+    "module-status-activated": "فعال شده",
+    "module-status-activegameplay": "فعال کردن گیم پلی",
+    "module-status-available": "در دسترس",
+    "module-status-dependency": "تبعیت",
+    "module-status-error": "Incompatible or unresolved dependencies",
+    "module-status-notpresent": "حاضر نیست",
+    "module-version-available": "آنلاین",
+    "module-version-installed": "نصب شده",
+    "module-version-installed-none": "هیچ",
+    "motion-blur": "حرکت ناشی از تیرگی",
+    "mouse-sensitivity": "حساسیت موشواره",
+    "movement-dead-zone": "منطقه ی مرده ی حرکت خط مرکزی",
+    "music-volume": "صدای موزیک",
+    "new-binding": "اتصال جدید",
+    "next-toolbar-item": "نمایش نوار ابزار بعدی",
+    "none": "هیچ",
+    "not-bound": "اماده نبودن",
+    "nui-editor-alternative-locale": "پیش دید جای چیزی",
+    "nui-editor-close": "بسته",
+    "nui-editor-copy": "کپی",
+    "nui-editor-disable-autosave": "قادر به ذخیره سازی خودکار نبودن",
+    "nui-editor-disable-icons": "قادر به نمایش تصویر نمودتر درختی نبودن",
+    "nui-editor-override": "ذخیره",
+    "nui-editor-paste": "انتقال متن",
+    "nui-editor-redo": "دوباره انجام دادن",
+    "nui-editor-save": "ذخیره سازی بعنوان...",
+    "nui-editor-settings-title": "ویرایشگر تنظیمات",
+    "nui-editor-undo": "خنثی کردن",
+    "oculus-rift-support": "پشتیبانی اوکلوس ریفت",
+    "open-inventory": "باز کردن موجودی",
+    "outline": "خارج شدن از نت",
+    "particle-effect-limit": "حداقل میزان تاثیر",
+    "parallax-mapping": "نمودار اختلاف منظر",
+    "parsing-content": "شکافتن محتویات ..",
+    "player-color": "رنگ بازیگر",
+    "player-name": "اسم بازیگر",
+    "player-height": "قد بازیگر",
+    "player-eye-height": "ارتفاع دید بازیگر",
+    "player-settings": "بازیگر",
+    "player-settings-title": "تنظیمات بازیگر",
+    "please-wait": "لطفا صبر کنید",
+    "preview-world-title": "پیش نمایش کلمه",
+    "preview-zoom-factor": "Zoom",
```

Comment:
This is untranslated - can you wrap this up just so the language is 100% finished? [8]